tests/openebs-pool-container-failure_test.go

```diff
@@ -166,4 +203,74 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 		})
 	})
 
+	//Matching the Resource Verison after Chaos
+	Context("Check Resource Version of pool container", func() {
+
+		It("Should check for the change in Resource Version after Chaos", func() {
+			rv2 := 0
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, v := range csp_rv.Items {
+				i2, _ := strconv.Atoi(v.ResourceVersion)
+				rv2 = rv2 + i2
+			}
+
+			if rv1 != rv2 {
```

Comment:
Instead of writing if-else you can use something like `Expect(rv1-rv2).NotTo(Equal(0))` . Use better name instead of rv1 & rv2 ( before, after)